### PR TITLE
Prepping representer for deploy

### PR DESCRIPTION
**Pull request by neenjaw**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 representer-*.tar
 
+/representer
+/bin/elixir_representer
```
Comment (neenjaw) [inline .gitignore:26]:
```suggestion
/bin/elixir_representer

```

File: bin/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mix deps.get
+mix deps.compile
+mix escript.build
+
+mv representer ./bin/elixir_representer
+chmod +x ./bin/elixir_representer
```
Comment (neenjaw) [inline bin/build.sh:8]:
```suggestion
chmod +x ./bin/elixir_representer

```

File: bin/generate.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+dir="$(dirname "${0}")"
+slug="${1}"
+solution_dir="${2}"
+
+printf "representing exercise %q in %q\n" "${slug}" "${solution_dir}"
+
+"${dir}/elixir_representer" "${slug}" "${solution_dir}"
```
Comment (neenjaw) [inline bin/generate.sh:9]:
```suggestion
"${dir}/elixir_representer" "${slug}" "${solution_dir}"

```

**Comment by neenjaw** [comment]
close #6

File: lib/representer/cli.ex [hunk 4]
```diff
@@ -1,5 +1,5 @@
 defmodule Representer.CLI do
-  @concatenated_file "concatenated.ex"
+  @concatenated_file "_concatenated.ex"
```
Comment (neenjaw) [inline lib/representer/cli.ex:2]:
```suggestion
  @concatenated_file "_concatenated.ex"
```

**Comment by neenjaw** [comment]
tagging as I can't directly add as reviewers:
@exercism/ops 
@ErikSchierboom

**Review by ErikSchierboom — APPROVED** [review]
LGTM!

**Pull request merged**